staging/src/k8s.io/cli-runtime/pkg/resource/builder_test.go

```diff
@@ -46,6 +46,7 @@ import (
 	"k8s.io/client-go/rest/fake"
 	restclientwatch "k8s.io/client-go/rest/watch"
 	"k8s.io/client-go/restmapper"
+
```

Comment:
nit: Redundant difference?

Reply:
Agree. This is done by gofmt. I would opt to leave this in as this will come up again the next time this file is touched.

Reply:
Just removed it.